wordpress-mobile/WordPress-iOS
# UL&S: Update the prologue buttons

Closes #14804 
Ref. #14009
Auth PR: https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/430

### To test:
1. Enable the `.unifiedWordPress` feature flag. 
2. `rake dependencies` then build & run.
3. The prologue screen should show two buttons that match the UL&S designs.
4. Tapping on "Continue with WordPress.com" navigates the user to the new Get Started view (aka unified WP.com login & signup flow). Go back.
4. Tapping on "Enter your site address" navigates the user to the unified Site Address flow. Complete the flow and dismiss the Epilogue.
5. Tap Me > Log In. The new prologue buttons should appear, as well as the Cancel button.

### Screenshots
**Prologue button design**
<img width="252" alt="Screen Shot 2020-08-31 at 11 57 01 AM" src="https://user-images.githubusercontent.com/1062444/91745750-17c02b80-eb81-11ea-9697-6d0246086b13.png">

| Light mode | Dark mode |
| --- | --- |
| ![Simulator Screen Shot - iPhone 11 Pro - 2020-08-31 at 11 59 34](https://user-images.githubusercontent.com/1062444/91747727-4db2df00-eb84-11ea-978c-85714eecfa8d.png) | ![Simulator Screen Shot - iPhone 11 Pro - 2020-08-31 at 12 19 18](https://user-images.githubusercontent.com/1062444/91747747-560b1a00-eb84-11ea-8741-380c80181a7b.png) |
| ![Simulator Screen Shot - iPhone 11 Pro - 2020-08-31 at 12 22 21](https://user-images.githubusercontent.com/1062444/91747893-a4b8b400-eb84-11ea-9697-45b09f6be274.png) | ![Simulator Screen Shot - iPhone 11 Pro - 2020-08-31 at 12 21 48](https://user-images.githubusercontent.com/1062444/91747914-ad10ef00-eb84-11ea-82d5-39a583e0746c.png) |

PR submission checklist:
